src/twisted/application/runner/test/test_runner.py

```diff
@@ -53,8 +62,8 @@ def setUp(self):
         # Patch globalLogBeginner so that we aren't trying to install multiple
         # global log observers.
 
-        self.stdout = BytesIO()
-        self.stderr = BytesIO()
+        self.stdout = StringIO()
+        self.stderr = StringIO()
```

Comment:
In the `sys` module, mypy says these are `TextIO` not `BytesIO`.

Reply:
The situation was real ugly when you had to take py2 into account but as of py3 it's simple and straightforward: stdio streams are text, if you want bytes, you have to access their `.buffer` attributes.  Even separate form mypy, In the current docs it's quite clearly laid out: https://docs.python.org/3.8/library/sys.html#sys.stdin

Reply:
Ah, I hadn't noticed that documentation, thanks